translations/ccs_household_gb_eng.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2020-05-01 12:54+0100\n"
+"POT-Creation-Date: 2020-05-01 20:01+0100\n"
```

Comment:
I wonder if we should be updating POT-Creation-Date if there are no changes to the translations, this will also lead to conflicts as seen on actions

Reply:
Accidental commit reverted.